[ENH] Document minimum cell size configuration, minor fixes (#261)
neurobagel/documentation: docs/user_guide/config.md

### Change security relevant variables

For security and best practice purposes, follow the below additional steps to configure your node if you are using a deployment profile that includes a graph store:

1. **Change the values** of the following variables in `.env` from their defaults:
    - `NB_GRAPH_USERNAME`
    - `NB_GRAPH_DB`

2. **Replace the default passwords** for the `admin` superuser and the newly created graph database user (`NB_GRAPH_USERNAME`) for your graph store with your own secure passwords. 

>     - These passwords are set using the contents of files named `NB_GRAPH_ADMIN_PASSWORD.txt` and `NB_GRAPH_PASSWORD.txt`, respectively. 
>     The default location of these files is [`./secrets`](https://github.com/neurobagel/recipes/tree/main/secrets) in the Neurobagel [`recipes`](https://github.com/neurobagel/recipes) repo.

Reviewer: ```suggestion
    - These passwords are stored in the directory defined under `{NB_GRAPH_SECRETS_PATH}` (`./secrets` by default)
    and are called `NB_GRAPH_ADMIN_PASSWORD.txt` and `NB_GRAPH_PASSWORD.txt` respectively. 
```

    - To generate a random password in the terminal, you can use:
      ```bash
      openssl rand -hex 16
      ```

>     - (Optional) Change the location of your password files to a more secure directory using the variable `NB_GRAPH_SECRETS_PATH`.

Reviewer: ```suggestion
    - (Optional) Change the directory where your password files are stored by editing the variable `NB_GRAPH_SECRETS_PATH` in your `.env` file.
```

>     ??? info "Graph store passwords are not meant for use by node query users"
>         The passwords specified in the deployment recipe are only used internally by the scripts that (automatically) set up and update the graph store, or to interact directly with the graph store (e.g., to modify database configuration or data).

Reviewer: ```suggestion
    ??? info "Graph store passwords are not meant for use by node query users"
        The passwords specified in the deployment recipe are only used internally by the scripts that (automatically) set up and update the graph store, 
        or to interact directly with the graph store (e.g., to modify database configuration or data).
        The passwords are also used to secure internal communication between your graph and its node API.
        An external query user is not able to query your graph directly.
```
Not sure this is clear yet. I think this could be a good clarification for folks confused by what the password does - but the phrasing is a bit odd. Maybe: "Graph store passwords are for internal communication only".
